Explorer/Assets/DCL/AvatarRendering/Emotes/Systems/Play/CharacterEmoteSystem.cs

```diff
@@ -243,6 +244,13 @@ private void CleanUp(Profile profile, in DeleteEntityIntention deleteEntityInten
                 messageBus.OnPlayerRemoved(profile.UserId);
         }
 
+        [Query]
+        private void DisableCharacterController(ref CharacterController characterController, in CharacterEmoteComponent emoteComponent)
+        {
+            bool isPlayingEmote = emoteComponent.CurrentAnimationTag == AnimationHashes.EMOTE || emoteComponent.CurrentAnimationTag == AnimationHashes.EMOTE_LOOP;
+            characterController.enabled = !isPlayingEmote;
```

Comment:
I am not sure it is the best way to prevent displacement. Did you test it on the moving platform?